scripts/e2e.sh

```diff
@@ -29,7 +29,7 @@ return-pod-exit-code() {
 
 wait-for-pod-ready() {
   local name="${1}"
-  local timeout_secs=180
+  local timeout_secs=15
```

Comment:
I originally at 180 seconds so I could test certain error conditions but it shoudl be at 15 seconds. Just seemed innocuous to go ahead and change this back to what it should be with this pr.